### ISSUE TITLE
Choix technologies

### ISSUE THREAD

**Issue by LucasWernerCESI** [LucasWernerCESI/station_meteo#6]
PYTHON (ESP), PHP (API/Connecteur SQL) et HTML/CSS/JS (appli web)
- [x] Prendre une déciion

**Comment by LucasWernerCESI** [comment]
Donc on part sur Python pour programmer l'ESP pour la transmission des données à l'API. L'API on la code aussi en Python tant qu'à faire ?
Et comme tout le projet tourne autour des requêtes HTTP autant utiliser uniquement HTML/CSS/JS pour la partie web, les communications entre l'appli Web et l'API se feront par requêtes AJAX.
D'autres idées ?

**Comment by NicoCesi789** [comment]
ça me convient

**Comment by LucasWernerCESI** [comment]
J'avais zappé mais du coup j'ai ajouté dans les features a prévoir, faudra créer un connecteur BDD avec des méthodes ou des fonctions qui permettent à l'API de toucher à la BDD. Même langage que l'API je pense.

**Comment by YannProject** [comment]
Yes ca me vas, nickel ! Pour l'API, Patrice nous a dit de pas réinventer la roue ... pourquoi ne pas en choper une et la modéliser a notre sauce ? ( simple suggestion =) )

**Comment by LucasWernerCESI** [comment]
Faut voir dans quelle mesure tu trouves une API qui fait ce que tu veux. Faut chercher API SCRUD REST PYTHON ou un truc du genre je pense. Perso je préfère concevoir le truc à partir de rien et chercher des morceaux de code qui feront ce que j'ai besoin de faire. A voir avec les autres.

**Comment by LucasWernerCESI** [comment]
Dans tous les cas Python pour la partie serveur/ESP et HTML/CSS/JS pour la partie appli web ca convient à tout le monde ?

**Comment by YannProject** [comment]
Yes ca me vas perso ! Ok pour l'API c'est une bonne idée au moins on sera exactement comment elle est faite

**Comment by LucieDesmons** [comment]
Ca me convient parfaitement !

**Comment by LucasWernerCESI** [comment]
Je déterre un peu mais pour le connecteur SQL et l'api Flask est apparemment particulièrement adapté.
